run_bin_test.go

```diff
@@ -578,6 +780,67 @@ func TestCoverageCollector_RunBinary(t *testing.T) {
 	}
 }
 
+func stdinPipePreFuncCovCollectorOption() CoverageCollectorOption {
+	f := PreCmdFunc(func(cmd *exec.Cmd) error {
+		writer, _ := cmd.StdinPipe()
+		_, err := writer.Write([]byte("Hello from prefunc\n"))
+		writer.Close()
+		return err
+	})
+	return PreExec(f)
+}
+
+func errPreCmdFuncCovCollectorOption() CoverageCollectorOption {
+	return PreExec(errPreCmdFunc())
+}
+
+func errPostCmdFuncCovCollectorOption() CoverageCollectorOption {
+	return PostExec(errPostCmdFunc())
+}
+
+func nilPreCmdFunc() CoverageCollectorOption {
+	f := PreCmdFunc(func(cmd *exec.Cmd) error {
+		return nil
+	})
+	return PreExec(f)
+}
+
+func printToBufferPreFunc(buffer *string) PreCmdFunc {
+	f := PreCmdFunc(func(cmd *exec.Cmd) error {
+		*buffer += "Hello from prefunc\n"
+		return nil
+	})
+	return f
+}
+
+func errPreCmdFunc() PreCmdFunc {
+	f := PreCmdFunc(func(cmd *exec.Cmd) error {
+		return errors.New("oh no!")
```

Comment:
Same here. Can pass in your error string as a param, and declare it as a const somewhere so you don't have to keep writing "oh no"